### ISSUE TITLE
Add SecurityToken support

### ISSUE THREAD

**Issue by patriclougheed** [humhub-contrib/onlydocuments#7]
There is no field in the options to include the token signature required for OnlyOffice to create a doc so I get an Invalid Token message. OnlyOffice v4.2+ now have security tokens that are required. 

https://api.onlyoffice.com/editors/signature/

![2017-10-17 11_24_51-dashboard - friendz - k-12 social network](https://user-images.githubusercontent.com/1463351/31682093-f5971b2e-b32d-11e7-9828-6c030be85967.jpg)

**Comment by trofim24** [comment]
Security tokens starting with version 4.2, but they are not required.

**Comment by patriclougheed** [comment]
Not required if you want your self hosted version left unprotected. A password is set by default and must be cleared if you want this module to work. As is, this does not work with a default install of OnlyOffice.

**Comment by ShockwaveNN** [comment]
@patriclougheed Seems that you use https://doc.onlyoffice.com like in example in configuration page.
So, we (ONLYOFFICE) enabled JWT security on https://doc.onlyoffice.com some time ago, so this server will not work with any setup (expect our, ONLYOFFICE).
You should setup our free `onlyoffice-documentserver` on your own server and connect to it. See: https://github.com/ONLYOFFICE/DocumentServer/

**Comment by patriclougheed** [comment]
Thanks but I am using my own install and it works great with OwnCloud because there is a setting to enter the security token. This module does not have a token field in the settings.

**Comment by Gaili** [comment]
Yeah, please add the possibility to work with the security token! Otherwise it is not secure to use the module. Thanks!

**Comment by marc-farre** [comment]
+1

**Comment by tovoro** [comment]
Are there any news regarding this issue?

**Comment by dreua** [comment]
I think this is fixed in the new version of this module and this issue can be closed: https://github.com/ONLYOFFICE/onlyoffice-humhub
